Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ambient"
-version = "0.2.7"
+version = "0.3.0"
```

Comment:
Think by semver it would be 0.2.8 (since there's no feature changes), but doesn't really matter

Reply:
I thought making it non-blocking could be considered a breaking change but I'll change it to 0.2.8